src/pyosmeta/contributors.py

```diff
@@ -1,9 +1,7 @@
 import json
```

Comment:
i am going to remove this file from this PR as we will handle that in another pr. this will break our contributor workflow as it is now